scylla/datadog_checks/scylla/metrics.py

```diff
@@ -0,0 +1,400 @@
+    'scylla_memory_streaming_dirty_bytes': 'memory.streaming_dirty_bytes',
+    'scylla_memory_streaming_virtual_dirty_bytes': 'memory.streaming_virtual_dirty_bytes',
+    'scylla_memory_system_dirty_bytes': 'memory.system_dirty_bytes',
+    'scylla_memory_system_virtual_dirty_bytes': 'memory.system_virtual_dirty_bytes',
+    'scylla_memory_total_memory': 'memory.total_memory',
+    'scylla_memory_virtual_dirty_bytes': 'memory.virtual_dirty_bytes',
+}
+
+SCYLLA_MEMTABLES = {
+    'scylla_memtables_pending_flushes': 'memtables.pending_flushes',
+    'scylla_memtables_pending_flushes_bytes': 'memtables.pending_flushes_bytes',
+}
+
+SCYLLA_NODE = {
+    'scylla_node_operation_mode': 'node.operation_mode',
+}
+
+SCYLLA_QUERY = {
+    'scylla_query_processor_queries': 'query_processor.queries',
+    'scylla_query_processor_statements_prepared': 'query_processor.statements_prepared',
+}
+
+SCYLLA_REACTOR = {
+    'scylla_reactor_aio_bytes_read': 'reactor.aio_bytes_read',
+    'scylla_reactor_aio_bytes_write': 'reactor.aio_bytes_write',
+    'scylla_reactor_aio_errors': 'reactor.aio_errors',
+    'scylla_reactor_aio_reads': 'reactor.aio_reads',
+    'scylla_reactor_aio_writes': 'reactor.aio_writes',
+    'scylla_reactor_cpp_exceptions': 'reactor.cpp_exceptions',
+    'scylla_reactor_cpu_busy_ms': 'reactor.cpu_busy_ms',
+    'scylla_reactor_cpu_steal_time_ms': 'reactor.cpu_steal_time_ms',
+    'scylla_reactor_fstream_read_bytes': 'reactor.fstream_read_bytes',
+    'scylla_reactor_fstream_read_bytes_blocked': 'reactor.fstream_read_bytes_blocked',
+    'scylla_reactor_fstream_reads': 'reactor.fstream_reads',
+    'scylla_reactor_fstream_reads_ahead_bytes_discarded': 'reactor.fstream_reads_ahead_bytes_discarded',
+    'scylla_reactor_fstream_reads_aheads_discarded': 'reactor.fstream_reads_aheads_discarded',
+    'scylla_reactor_fstream_reads_blocked': 'reactor.fstream_reads_blocked',
+    'scylla_reactor_fsyncs': 'reactor.fsyncs',
+    'scylla_reactor_io_queue_requests': 'reactor.io_queue_requests',
+    'scylla_reactor_io_threaded_fallbacks': 'reactor.io_threaded_fallbacks',
+    'scylla_reactor_logging_failures': 'reactor.logging_failures',
+    'scylla_reactor_polls': 'reactor.polls',
+    'scylla_reactor_tasks_pending': 'reactor.tasks_pending',
+    'scylla_reactor_tasks_processed': 'reactor.tasks_processed',
+    'scylla_reactor_timers_pending': 'reactor.timers_pending',
+    'scylla_reactor_utilization': 'reactor.utilization',
+}
+
+SCYLLA_SCHEDULER = {
+    'scylla_scheduler_queue_length': 'scheduler.queue_length',
+    'scylla_scheduler_runtime_ms': 'scheduler.runtime_ms',
+    'scylla_scheduler_shares': 'scheduler.shares',
+    'scylla_scheduler_tasks_processed': 'scheduler.tasks_processed',
+    'scylla_scheduler_time_spent_on_task_quota_violations_ms': 'scheduler.time_spent_on_task_quota_violations_ms',
+}
+
+SCYLLA_SCYLLADB = {
+    'scylla_scylladb_current_version': 'scylladb.current_version',
```

Comment:
This will never be collected since it's defined as the `metadata_metric_name`, so you can remove this

Reply:
Cool, will remove that group.